playground/TestPlatform.Playground/Program.cs

```diff
@@ -44,9 +44,17 @@ static void Main()
                 <RunConfiguration>
 
                     <!-- <MaxCpuCount>1</MaxCpuCount> -->
+                    <!-- <DisableParallelization>True<DisableParallelization> -->
                     <!-- <TargetPlatform>x86</TargetPlatform> -->
                     <!-- <TargetFrameworkVersion>net472</TargetFrameworkVersion> -->
 
+                    <!-- Per test coverage support -->
+                    <!--
+                    <MaxCpuCount>0</MaxCpuCount>
+                    <ForceOneTestAtTimePerTestHost>True</ForceOneTestAtTimePerTestHost>
+                    <TargetFrameworkTestHostLoadBalancingStrategy>4</TargetFrameworkTestHostLoadBalancingStrategy>
```

Comment:
Woopsie :) This naming suggestion was for keeping your strategy idea. 

Hum, I don't have super good name popping up in my mind... Maybe your multi-host suggestion was not bad... Ideas: `SpecificMultiTestHostCount`, `TargetFrameworkTestHostDemultiplexer`?